### PR TITLE
11-Eliminating-Distractions ibt

### PR DESCRIPTION
**Pull request by ibtehajraza**
Hi there, as per your requirement I'm sending you this pull request. Please review this.

### CODE REVIEW

File: content/post/11-Eliminating-Distractions/index.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Eliminating Distractions"
+date: 2010-11-13T15:56:33-07:00
```
Comment (josh-padnick) [inline content/post/11-Eliminating-Distractions/index.md]: Now that I think about it, we may as well set the seconds to zero, so can you adjust this date as follows?

```suggestion
date: 2010-11-13T00:00:00-07:00
```

File: content/post/11-Eliminating-Distractions/index.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+draft: true
```
Comment (josh-padnick) [inline content/post/11-Eliminating-Distractions/index.md]: We want the new posts to be ready for publication, so please set this to false on all posts:

```suggestion
draft: false
```

File: content/post/11-Eliminating-Distractions/index.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+description: This is a description
```
Comment (josh-padnick) [inline content/post/11-Eliminating-Distractions/index.md]: This is mistake on my part. Can you delete the `description` property here and on all the future posts you add?

File: content/post/11-Eliminating-Distractions/index.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+categories:
+- life
+- travel
```
Comment (josh-padnick) [inline content/post/11-Eliminating-Distractions/index.md]: If you look at the bottom-right hand corner of the original blog post on my current site, it shows the only category as "zen." Can you add that using all lowercase? 

image

So, this should be changed to:

```yaml
categories:
- zen
```

File: content/post/11-Eliminating-Distractions/index.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+Well, evolution has given us many wonderful things, but as my man Thoreau once said “You never gain something but that you lose something.”  In other words, when humanity discovered the car, we gained all kinds of wonderful things like being able to travel long distances, but we also lost the joy of just walking to get where we needed to.
+
+And so it is with the Internet.  We gain incredible knowledge, access, communication, and most amazing of all rapid sharing of information and ideas.  But now we’ve lost the simplicity that prior generations had.  Distractions are literally assaulting you all day long.
+
+The purpose of this post is just to admit the problem and link to a new book by Leo Babauta (whom I’ve previously read and enjoyed) on the subject of simplicity in a world full of distractions.  The book is simply called “Focus” and you [download a PDF](http://zenhabits.net/focus-book/) that is completely free (and amazingly uncopyrighted).
```
Comment (josh-padnick) [inline content/post/11-Eliminating-Distractions/index.md]: You copied this correctly, but I realized I had a typo:

```suggestion
The purpose of this post is just to admit the problem and link to a new book by Leo Babauta (whom I’ve previously read and enjoyed) on the subject of simplicity in a world full of distractions.  The book is simply called “Focus” and you should [download a PDF](http://zenhabits.net/focus-book/) that is completely free (and amazingly uncopyrighted).
```

File: content/post/11-Eliminating-Distractions/index.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+description: This is a description
+categories:
+- life
+- travel
+
+---
+
+When man’s primary job was to find a way to eat each day, distractions were probably not a big deal.  Primitive man had no facebook, no twitter, no IM, no cell phone.  He just had a rumbling in his stomach and the grim realization that either he found some food or he and possibly his family wouldn’t make it past winter.
+
+Well, evolution has given us many wonderful things, but as my man Thoreau once said “You never gain something but that you lose something.”  In other words, when humanity discovered the car, we gained all kinds of wonderful things like being able to travel long distances, but we also lost the joy of just walking to get where we needed to.
```
Comment (josh-padnick) [inline content/post/11-Eliminating-Distractions/index.md]: You copied this correctly, but here's a minor wording improvement:

```suggestion
Well, evolution has given us many wonderful things, but as Thoreau once said “You never gain something but that you lose something.”  In other words, when humanity discovered the car, we gained all kinds of wonderful things like being able to travel long distances, but we also lost the joy of just walking to get where we needed to.
```

**Review by josh-padnick — COMMENTED** [review]
@ibtehajraza This looks great overall. Thank you! In addition to my comments, the one other point of feedback I'd like to give is that the file itself should be named:

```
 content/post/11-eliminating-distractions/index.md
```

That is, use all lower case in the name. I'll respond separately to you via Upwork.

File: content/post/11-Eliminating-Distractions/index.md (hunk 6 above)
Comment (josh-padnick) [inline content/post/11-Eliminating-Distractions/index.md]: Can you add the `<!--more-->` after this line and before the previous line? This allow the home page to summarize the article correctly.

Comment (josh-padnick) [reply content/post/11-Eliminating-Distractions/index.md]: Note that you'll need to add these on all the blog posts unless the post is especially short. For guidance on a cutoff point, see the current site.

**Comment by ibtehajraza** [comment]
well sure let me fix these bugs and make a pull request.

**Comment by josh-padnick** [comment]
Please see my previous comment on `<!--more-->`. Also, can you use all lowercase on the file name so that it's called ` content/post/11-eliminating-distractions/index.md`? Once those are completed, this will be ready to merge.

**Comment by josh-padnick** [comment]
Alright, this is approved! I'll merge now. :) Feel free to get started on the remaining posts. Opening one Pull Request per blog post seems like a clean way to review things.

**Pull request merged**